superflore/generators/ebuild/run.py

```diff
@@ -116,34 +116,48 @@ def main():
                     overlay.repo.get_last_hash()
                 )
             )
+            missing_depends = list()
```

Comment:
Are these order dependent and support repeating? Otherwise it should be a set for clearer semantics.

Reply:
For sure -- I agree this should be a set.